DNN Platform/Library/Services/Installer/Installers/CleanupInstaller.cs

```diff
@@ -75,7 +75,7 @@ private bool ProcessCleanupFile()
             }
             catch (Exception ex)
             {
-                Log.AddFailure(ex);
+                Log.AddWarning(string.Format(Util.CLEANUP_ProcessComplete, ex.Message));
```

Comment:
is this supposed to be using the `CLEANUP_ProcessError` field you defined below?

Reply:
Yes. I'll adjust